scripts/variantstore/wdl/GvsQuickstartIntegration.wdl

```diff
@@ -64,7 +64,6 @@ workflow GvsQuickstartIntegration {
             input_vcfs = input_vcfs,
             input_vcf_indexes = input_vcf_indexes,
             filter_set_name = "quickit",
-            create_filter_set_scatter_count = 20,
```

Comment:
does this mean we want to remove it from the quickstart instructions?

Reply:
good idea — done